### DISCUSSION TITLE
Как пропустить вход в систему напрямую, просто чтобы протестировать и испытать этот проект？

### DISCUSSION THREAD

**Discussion by youthyan** [midori-no-me/ichime#73] [Q&A]
Я считаю, что этот проект замечательный. После того как я собрал этот проект, для входа на главную страницу требуется учетная запись на сайте anime365. Мне это интересно, но у меня нет учетной записи, и я не могу войти. Я просто хочу испытать функциональность без входа в учетную запись или пропустить вход, например, как пропустить вход, чтобы отображались только изображения и данные заголовков, или воспроизвести трейлер для тестирования. Я хочу знать, какую часть кода нужно изменить, или есть ли другие способы, чтобы я мог испытать этот замечательный проект. Спасибо.

**Comment by flaksp** [comment]
Необходимо создать аккаунт на https://anime365.ru и использовать эти данные для входа. Обратите внимание, что для просмотра видео (в том числе и трейлеров) требуется активная [платная подписка](https://anime365.ru/support/index) на сайте. Приложение не поддерживает работу без аккаунта именно потому, что большая часть возможностей сайта требует платной подписки.

Вроде как в приложение возможно войти, не имея платную подписку, но я не помню, чтобы мы это тестировали, поэтому без наличия подписки оно может работать некорректно. Ну и видео в таком случае, разумеется, загружаться не будут.

**Comment by youthyan** [comment]
На сайте нет аккаунта для тестирования? Например, у него есть только одно тестовое видео,
Или есть простой способ проверить это?
Должен ли я платить, если хочу протестировать?

**Comment by flaksp** [comment]
К сожалению, пробного периода у сайта нет, поэтому потребуется оформление платной подписки. Тестовых видео на сайте тоже нет.
